crates/router/src/core/refunds.rs

```diff
@@ -970,7 +977,7 @@ pub async fn add_refund_execute_task(
         .change_context(errors::ApiErrorResponse::InternalServerError)
         .attach_printable_lazy(|| format!("unable to convert into value {:?}", &refund))?;
     let process_tracker_entry = storage::ProcessTrackerNew {
-        id: format!("{}_{}_{}", runner, task, refund.id),
+        id: format!("{}_{}_{}", runner, task, refund.id), // shouldn't merchant_id be added here.
```

Comment:
added this in process tracker PR https://github.com/juspay/hyperswitch/pull/1359